src/sentry/api/endpoints/project_rules.py

```diff
@@ -104,24 +452,30 @@ def get(self, request: Request, project) -> Response:
             on_results=lambda x: serialize(x, request.user),
         )
 
+    @extend_schema(
+        operation_id="Create an Issue Alert Rule for a Project",
+        parameters=[
+            GlobalParams.ORG_SLUG,
+            GlobalParams.PROJECT_SLUG,
+        ],
+        request=ProjectRulesPostSerializer,
+        responses={
+            201: inline_sentry_response_serializer("RuleCreated", RuleSerializerResponse),
```

Comment:
This should be the regular serializer over the TypedDict when the response is not a list
```suggestion
            201: RuleSerializer,
```